src/reload_save.c

```diff
@@ -14,7 +14,6 @@ void ReloadSave(void)
 {
     u16 imeBackup = REG_IME;
     REG_IME = 0;
-    RegisterRamReset(RESET_EWRAM);
```

Comment:
How come we don't need this anymore? I'd have thought maybe we should clear the RAM and then reload the initialized variables?

Reply:
You're right, we need this. I thought it wouldn't make sense to clear everything, but really we need to clear everything in EWRAM and then re-init. I'm having trouble jumping into the ARM assembly from the THUMB main -- every time I do I get an exception vector firing when I try to load from the LMA. Need to figure that out, but it should work in theory.

Reply:
Converting to draft until I can figure out why calling `InitializeWorkingMemory` (previously the `RamFill` in crt0) from C crashes the hardware. If you or anyone else is able to take a look as well I'd be grateful. Not too familiar with calling convention on this platform.

Reply:
I got it tested and working, the trick was just re-write it in thumb for the re-init.